yml/apdb.yaml

```diff
@@ -2802,6 +2802,11 @@ tables:
     datatype: double
     description: Declination of detector center.
     fits:tunit: deg
+  - name: pixelScale
+    '@id': '#DetectorVisitProcessingSummary.pixelScale'
+    datatype: float
+    description: Measured detector pixel scale (arcsec/pixel).
+    fits:tunit:
```

Comment:
I believe this could be `arcsec/pixel` here. (That is a valid unit.)

Reply:
Oh, really?  When I first tried looking into this, it didn't seem to be...very happy to hear it is!  I'll update this everywhere.